scripts/dotnet.py

```diff
@@ -78,12 +78,10 @@ def get_target_framework_moniker(framework: str) -> str:
         To run NativeAOT benchmarks we need to run the host BDN process as latest
         .NET the host process will build and run AOT benchmarks
         '''
-        if framework == 'nativeaot6.0':
-            return 'net6.0'
-        if framework == 'nativeaot7.0':
-            return 'net7.0'
-        if framework == 'nativeaot8.0':
-            return 'net8.0'
+        pattern = r'^nativeaot(\d+)\.0$'
```

Comment:
Although this makes it more general, I think the readability is not great.

Reply:
That makes sense, decided to continue the manual check for readability and consistency.